lib/oli/help/providers/freshdesk_help.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Oli.Help.Providers.FreshdeskHelp do
+  @behaviour Oli.Help.Dispatcher
+
+  alias Oli.Help.HelpContent
+
+  @headers [
+    {"Content-type", "application/json"},
+    {"Accept", "application/json"},
+    {"Authorization", "Basic " <> Base.encode64(System.get_env("FRESHDESK_API_KEY", "examplekey"))}
+  ]
+
+  @impl Oli.Help.Dispatcher
+  def dispatch(%HelpContent{} = contents) do
+    url = System.get_env("FRESHDESK_API_URL", "example.edu")
+
+    {:ok, body} =
```

Comment:
It would be more robust here to handle {:error, error} clause and return an {:error, message}, as opposed to letting this process die.